source/NationalInstruments/jquery.flot.scattergraph.js

```diff
@@ -516,6 +517,40 @@ THE SOFTWARE.
 
                 ctx.restore();
             };
+
+            function findNearbyItems (plot, canvasX, canvasY, series, seriesIndex, radius, computeDistance, items) {
+                var ps = series[seriesIndex].datapoints.pointsize;
+                for (var j = 0; j < series[seriesIndex].data.length; j++) {
+                    for (var i = 0; i < dataLen; i++) {
+                        var x, y;
+                        if (isArrayOfObject) {
+                            x = series[seriesIndex].data[j][i].x;
+                            y = series[seriesIndex].data[j][i].y;
+                        } else if (isObjectOfArray) {
+                            x = series[seriesIndex].data[j].x[i];
+                            y = series[seriesIndex].data[j].y[i];
+                        } else if (isArrayOfArray) {
+                            x = series[seriesIndex].data[j][i][0];
+                            y = series.data[j][i][1];
+                        }
+
+                        var px = series[seriesIndex].xaxis.p2c(x);
+                        var deltaX = px - canvasX;
+                        var py = series[seriesIndex].yaxis.p2c(y);
+                        var deltaY = py - canvasY;
+                        var distance = Math.sqrt(deltaX * deltaX + deltaY * deltaY);
```

Comment:
Seems like we might want to use the `computeDistance` param here if it's defined.

Reply:
done